package.json

```diff
@@ -186,7 +186,8 @@
         }
     },
     "dependencies": {
-        "@hubspot/cli-lib": "3.0.4",
+        "@hubspot/cli": "^3.0.7-beta.5",
+        "@hubspot/cli-lib": "^3.0.7-beta.5",
```

Comment:
When https://github.com/HubSpot/hubspot-cli/pull/534/files is merged, will pin to newest release